src/utils/isReactComponentClass.js

```diff
@@ -44,6 +44,26 @@ export default function isReactComponentClass(
     return true;
   }
 
+  // check for @extends React.Component in docblock
+  if (path.parentPath.value && path.parentPath.value) {
+    var classDeclaration = {};
+    if (Array.isArray(path.parentPath.value)) {
+      var matches = path.parentPath.value.filter(function(declaration) { return declaration.type === 'ClassDeclaration' });
+      if (matches[0]) {
+        classDeclaration = matches[0];
+      }
+    } else {
+      classDeclaration = path.parentPath.value;
+    }
+    
+    if (classDeclaration.leadingComments && classDeclaration.leadingComments.length > 0) {
```

Comment:
Be more defensive and add a check for whether `classDeclaration` is truthy:

```js
if (classDeclaration && classDeclaration.leadingComments && classDeclaration.leadingComments.length) {
```

(Note that checking the truthiness of `length` is sufficient rather than doing the explicit `> 0`)